metrics/default_metrics/tarantool/psutils_linux.lua

```diff
@@ -0,0 +1,70 @@
+local fio = require('fio')
+local string = require('string')
+local ffi = require('ffi')
+local log = require('log')
+
+ffi.cdef[[
+    int get_nprocs_conf(void);
+]]
+
+local function get_cpu_time()
+    local cpu_stat_file = fio.open('/proc/stat', 'O_RDONLY')
+    if cpu_stat_file == nil then
+        log.error('/proc/stat open error')
```

Comment:
Not sure we should something log inside metrics. It's a simple way to spam logs.

Reply:
agreed 
must be no log calls